src/components/draw/DrawDirective.js

```diff
@@ -110,6 +110,9 @@ goog.require('ga_permalink');
       var updateHelpTooltip = function(overlay, type, drawStarted, onFirstPoint,
           onLastPoint) {
         var helpMsg = $translate.instant('draw_start_' + type);
+        if (type == 'marker' || type == 'annotation') {
+          helpMsg = $translate.instant('draw_next_' + type);
```

Comment:
When you add a point there is no next message,  the help message is always "Add a point". The translation exists but it's never used so we could remove it, I guess.